argocd/overlays/moc-infra/applications/envs/moc/smaug/chris/chris.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: chris
+spec:
+  destination:
+    name: smaug
+    namespace: chris-system
```

Comment:
i dont think we have this namespace

please select any of the below
chrisproject
chrisproject-dev
chrisproject-staging